CONTRIBUTING.md

```diff
@@ -3,13 +3,12 @@
 We are beyond excited to see that you want to contribute! We would love to accept your contributions. Layer5 is built by the community and warmly welcomes collaboration. There are many ways in which one could contribute to Layer5 and every contribution is equally appreciated here. Navigate through the following to understand more about contributing here. 
 
   - [Before You Get Started](#before-you-get-started)
-  - [Contributing to Layer5's Code](#contributing-to-layer5s-code)
-  - [Contributing to Layer5's Docs](#contributing-to-layer5s-docs)
+  - [Contributing to Layer5 Projects](#contributing-to-layer5-projects)
+  - [Contributing to Layer5's Docs](#contributing-to-layer5s-blogs)
```

Comment:
The hyperlink works as-is, correct? This change seems to break the link. 

The title of the link can be updated from "Docs" to "Blogs".